common/docker-entrypoint.sh

```diff
@@ -1,25 +1,78 @@
 #!/bin/bash
-set -eu
+set -e
+
+# DEBUG flag to enable debug output. To turn on debug set DEBUG=1 (e.g. docker compose up -d --build --env DEBUG=1)
+TEMP_DEBUG="${DEBUG:-0}"
+if [[ "$TEMP_DEBUG" == "1" || "${TEMP_DEBUG}" == "true" || "${TEMP_DEBUG}" -eq 1 ]]; then
+  echo "Startup debug logging enabled."
+  DEBUG=1
+else
+  DEBUG=0
+fi
+export DEBUG
+
+# default the db port to 3306 if not set
+export MAUTIC_DB_PORT="${MAUTIC_DB_PORT:-3306}"
 
 # set a few variables to be used in the entrypoint scripts
 export MAUTIC_VOLUME_CONFIG="${MAUTIC_VOLUME_CONFIG:-/var/www/html/config}"
-export MAUTIC_VOLUME_CONSOLE="${MAUTIC_VOLUME_CONFIG:-/var/www/html/bin/console}"
 export MAUTIC_VOLUME_LOGS="${MAUTIC_VOLUME_LOGS:-/var/www/html/var/logs}"
 export MAUTIC_VOLUME_MEDIA="${MAUTIC_VOLUME_MEDIA:-/var/www/html/docroot/media}"
 
+export MAUTIC_VAR="${MAUTIC_VAR:-/var/www/html/var}"
+export MAUTIC_CONSOLE="${MAUTIC_CONSOLE:-/var/www/html/bin/console}"
+
 export MAUTIC_WWW_USER="${MAUTIC_WWW_USER:-www-data}"
 export MAUTIC_WWW_GROUP="${MAUTIC_WWW_GROUP:-www-data}"
 
 
+# TODO consider adding "var/*" as volumes checks for future because they might need to exist but also have correct perms
+export MAUTIC_VOLUMES="\
+${MAUTIC_VOLUME_CONFIG} \
+${MAUTIC_VAR} \
+${MAUTIC_VOLUME_LOGS} \
+${MAUTIC_VOLUME_MEDIA}"
+
+export REQUIRED_MAUTIC_VARIABLES="\
+MAUTIC_DB_HOST \
+MAUTIC_DB_USER \
+MAUTIC_DB_PASSWORD \
+DOCKER_MAUTIC_ROLE"
```

Comment:
```suggestion
DOCKER_MAUTIC_ROLE \
MAUTIC_DB_DATABASE"
```

It's important to also define "MAUTIC_DB_DATABASE"

Reply:
MAUTIC_DB_DATABASE has a default in app and is not required as a defined variable